moveit_ros/planning_interface/move_group_interface/include/moveit/move_group_interface/move_group_interface.h

```diff
@@ -48,7 +48,9 @@
 #include <moveit_msgs/Grasp.h>
 #include <moveit_msgs/PlaceLocation.h>
 #include <moveit_msgs/MotionPlanRequest.h>
+#include <moveit_msgs/MoveGroupAction.h>
 #include <geometry_msgs/PoseStamped.h>
+#include <actionlib/client/simple_action_client.h>
```

Comment:
This line makes `actionlib` a `CATKIN_DEPENDS` of the `moveit_ros_planning_interface` package,
i.e. packages including the header have to be able to find actionlib headers.

Can you please add that to the `CMakeLists.txt`?

Reply:
Done.